app/Config/core.php

```diff
@@ -104,36 +105,36 @@
  * included primarily as a development convenience - and
  * thus not recommended for production applications.
  */
-	//Configure::write('App.baseUrl', env('SCRIPT_NAME'));
+    //Configure::write('App.baseUrl', env('SCRIPT_NAME'));
 
-/**
+/*
  * To configure CakePHP to use a particular domain URL
  * for any URL generation inside the application, set the following
  * configuration variable to the http(s) address to your domain. This
  * will override the automatic detection of full base URL and can be
  * useful when generating links from the CLI (e.g. sending emails)
  */
-	//Configure::write('App.fullBaseUrl', 'http://example.com');
+    //Configure::write('App.fullBaseUrl', 'http://example.com');
 
-/**
+/*
  * Web path to the public images directory under webroot.
  * If not set defaults to 'img/'
  */
-	//Configure::write('App.imageBaseUrl', 'img/');
+    //Configure::write('App.imageBaseUrl', 'img/');
```

Comment:
__[staging][code_sniffer]__
* Comment closer must be on a new line
* No space found before comment text; expected "// Configure::write('App.imageBaseUrl', 'img/');" but found "//Configure::write('App.imageBaseUrl', 'img/');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment